weasyl/moderation.py

```diff
@@ -470,74 +470,57 @@ def setusermode(userid, form):
-
+def submissionsbyuser(targetid):
     query = d.engine.execute("""
-        SELECT su.submitid, su.title, su.rating, su.unixtime, su.userid, pr.username, su.settings
+        SELECT su.submitid, su.title, su.rating, su.unixtime, su.settings
         FROM submission su
-            INNER JOIN profile pr USING (userid)
-        WHERE su.userid = (SELECT userid FROM login WHERE login_name = %(sysname)s)
+        WHERE su.userid = %(user)s
         ORDER BY su.submitid DESC
-    """, sysname=d.get_sysname(form.name))
+    """, user=targetid)
 
     ret = [{
         "contype": 10,
+        "userid": targetid,
         "submitid": i[0],
         "title": i[1],
         "rating": i[2],
         "unixtime": i[3],
-        "userid": i[4],
-        "username": i[5],
-        "settings": i[6],
+        "settings": i[4],
     } for i in query]
     media.populate_with_submission_media(ret)
     return ret
 
 
-def charactersbyuser(userid, form):
-    if userid not in staff.MODS:
-        raise WeasylError("Unexpected")
-
+def charactersbyuser(targetid):
     query = d.engine.execute("""
         SELECT
-            ch.charid, pr.username, ch.unixtime,
-            ch.char_name, ch.age, ch.gender, ch.height, ch.weight, ch.species,
-            ch.content, ch.rating, ch.settings, ch.page_views, pr.config
+            ch.charid, pr.username, ch.unixtime, ch.char_name, ch.rating,
+            ch.settings
         FROM character ch
-        INNER JOIN profile pr ON ch.userid = pr.userid
-        INNER JOIN login ON ch.userid = login.userid
-        WHERE login.login_name = %(sysname)s
-    """, sysname=d.get_sysname(form.name))
+            INNER JOIN profile USING (userid)
+        WHERE ch.userid = %(user)s
+    """, user=targetid)
 
     return [{
         "contype": 20,
-        "userid": userid,
+        "userid": targetid,
         "charid": item[0],
-        "username": item[1],
         "unixtime": item[2],
         "title": item[3],
-        "rating": item[10],
-        "settings": item[11],
-        "sub_media": character.fake_media_items(item[0], userid, d.get_sysname(item[1]), item[11]),
+        "rating": item[4],
+        "settings": item[5],
+        "sub_media": character.fake_media_items(item[0], targetid, d.get_sysname(item[1]), item[5]),
     } for item in query]
 
 
-def journalsbyuser(userid, form):
-    if userid not in staff.MODS:
-        raise WeasylError("Unexpected")
-
-    results = d.engine.execute("""
-        SELECT journalid, title, journal.settings, journal.unixtime, rating,
-               profile.username, 30 contype
-          FROM journal
-               JOIN profile USING (userid)
-               JOIN login USING (userid)
-         WHERE login_name = %(sysname)s
-    """, sysname=d.get_sysname(form.name)).fetchall()
+def journalsbyuser(targetid):
+    query = d.engine.execute("""
+        SELECT journalid, title, settings, unixtime, rating
+        FROM journal
+        WHERE jo.userid = %(user)s
```

Comment:
```suggestion
        WHERE userid = %(user)s
```

And here it looks like the ``jo`` got left behind.

https://paste.weasyldev.com/show/bSlGk3PwosxSZivsmnUC/
```
2020-02-21T01:06:07+0000 [stderr#error] ProgrammingError: (psycopg2cffi._impl.exceptions.ProgrammingError) missing FROM-clause entry for table "jo"
2020-02-21T01:06:07+0000 [stderr#error] LINE 4:         WHERE jo.userid = 36064
2020-02-21T01:06:07+0000 [stderr#error]                       ^
2020-02-21T01:06:07+0000 [stderr#error] [SQL:
2020-02-21T01:06:07+0000 [stderr#error]         SELECT journalid, title, settings, unixtime, rating
2020-02-21T01:06:07+0000 [stderr#error]         FROM journal
2020-02-21T01:06:07+0000 [stderr#error]         WHERE jo.userid = %(user)s
2020-02-21T01:06:07+0000 [stderr#error]     ]
```